.github/scripts/spellcheck.py

```diff
@@ -0,0 +1,48 @@
+import os
+from spellchecker import SpellChecker
+import json
+
+# Initialize spell checker
+spell = SpellChecker()
+
+# List of directories to scan for files
+directories = ["_data", "utils"]  # Scan only _data and utils directories
```

Comment:
We need to review other directories too. eg:- swan-lake, downloads, case-studies etc.

Reply:
@sm1990 can you please merge the PR and adjust it based on your needs... 
the task was to created the action and it works please adjust it on your needs!

Reply:
```suggestion
directories = ["_data", "case-studies", "community", "components", "downloads", "hacktoberfest", "pages", "policy", "spec", "swan-lake", "utils"] 
```

Reply:
added